atm.rb

```diff
@@ -1,13 +1,64 @@
+# ATM by Eric Anderson for evaluation
 class Atm
-  STARTING_BALANCE = 100.0
-
-  def initialize(filename = "balance.txt")
+  def initialize(filename = 'balance.txt')
     @filename = filename
   end
 
   def run
-    balance = STARTING_BALANCE
```

Comment:
You removed the constant, which was required in the specifications:
`If the file does not exist, use a starting balance of 100.0, which must be a constant.`